app/api_alpha/views.py

```diff
@@ -1162,15 +1162,28 @@ def get(self, request):
                     COPY (
                         select
                         CASE
-                            WHEN event_type = 'deactivation' THEN 'delete'
+                            WHEN event_type = 'delete' THEN 'deactivate'
+                            WHEN event_type = 'deactivation' THEN 'deactivate'
+                            WHEN event_type = 'deactivation' THEN 'deactivate'
```

Comment:
```suggestion
                            WHEN event_type = 'deactivation' THEN 'deactivate'
```

Il n'y aura plus de `delete` dès ce soir si tout va bien

Reply:
Je viens de vérifier et on a encore des `delete` en base. Je supprime la ligne en doublon mais je conserve celle prenant en compte `delete`